testsuite/unittests/test_gap_junction.sli

```diff
@@ -82,7 +98,7 @@ neuron_no_gap neuron_no_gap Connect
 0 /max_delay get /old_max Set
 0 /min_delay get /old_min Set
 
-neuron_gap neuron_gap 4.0 4.0 /gap_junction Connect
+neuron_no_gap neuron_no_gap 2.0 2.0 Connect
```

Comment:
@jakobj I think you understood me wrong, I did not consider the code obsolete. 

@janhahne I think things got messed up in the merging process. I don't fully understand what is to be tested here. My current interpretation of the code is that you first set `wfr_comm_interval` to 2.0 ms. Then you create one connection with default delay (1 ms), to provoke an update of min_delay and max_delay. This should result in min_delay = 1 ms and max_delay = 2 ms. Then you create another connection between normal neurons, with explicit delay, and check again min_delay and max_delay. For that second test to make sense, this last connect needs to use a delay smaller than 1 ms or larger than 2 ms, so the line should probably be
```
neuron_no_gap neuron_no_gap 1.0 4.0 Connect
```